UserDetails/models.py

```diff
@@ -35,39 +41,8 @@ def can_access_back(self):
 
 
 class Association(Group):
-    class Meta:
```

Comment:
Dat moment dat je ziet dat Association nog een proxy is terwijl dat al lang niet handig meer is XD Goede fix.